website/pages/docs/k8s/helm.mdx

```diff
@@ -870,10 +872,104 @@ and consider if they're appropriate for your deployment.
+
+    - `tolerations` ((#v-ingressgateways-defaults-tolerations)) (`string: null`) - Optional YAML string to specify tolerations.
+
+    - `nodeSelector` ((#v-ingressgateways-defaults-nodeselector)) (`string: null`) - Optional YAML string to specify nodeSelector config.
+
+    - `priorityClassName` ((#v-ingressgateways-defaults-priorityclassname)) (`string: ""`) - Optional priorityClassName.
+
+    - `annotations` ((#v-ingressgateways-defaults-annotations)) (`string: null`) - Annotations for the ingress gateway deployment. Annotations defined here will be applied to all ingress gateway deployments in addition to any annotations defined for a specific gateway in `ingressGateways.gateways`.
+
+      ```yaml
+      annotations: |
+        "annotation-key": "annotation-value"
+      ```
+
+    - `consulNamespace` ((#v-ingressgateways-defaults-consulnamespace)) (`string: default`) <EnterpriseAlert inline /> - Defines the Consul namespace to register the gateway into.  Requires `global.enableConsulNamespaces` to be true and
+    Consul Enterprise v1.7+ with a valid Consul Enterprise license. Note: The Consul namespace MUST exist before the gateway is deployed.
+
+
+  - `gateways` ((#v-ingressgateways-gateways)) - Gateways is a list of gateway objects. The only required field for each is `name`, though they can also contain any of the fields in `ingressGateways.defaults`. Values defined here override the defaults except in the case of annotations where both will be applied.
+
+    - `name` ((#v-ingressgateways-gateways-name)) (`string: "ingress-gateway"`) - The name of the ingress gateway.
+
+- `terminatingGateways` ((#v-terminatinggateways)) - Configuration options for terminating gateways. Default values for all terminating gateways are defined in `terminatingGateways.defaults`. Any of these values may be overridden in `terminatingGateways.gateways` for a specific gateway with the exception of annotations. Annotations will include both the default annotations and any additional ones defined for a specific gateway. Requirements: consul >= 1.8.0. If using global.acls.manageSystemACLs, consul-k8s >= 0.16.0 is needed.
+
+  - `enabled` ((#v-terminatinggateways-enabled)) (`bool: false`) - Enable terminating gateway deployment. Requires `connectInject.enabled=true`.
+
+  - `defaults` ((#v-terminatinggateways-defaults)) - Defaults sets default values for all gateway fields. With the exception of annotations, defining any of these values in the `terminatingGateways.gateways` list will override the default values provided here.
+
+    - `replicas` ((#v-terminatinggateways-defaults-replicas)) (`int: 2`) - Number of replicas for each terminating gateway defined.
+
+    - `extraVolumes ((#v=terminatinggateways-defaults-extraVolumes)) (`array: []`) - A list of extra volumes to mount. These will be exposed to Consul in the path `/consul/userconfig/<name>/`.
```

Comment:
I think this might be causing the website build error https://app.netlify.com/sites/consul-docs-preview/deploys/5ee9677acee1cf00072a470e. Maybe try `{name}`?

```
5:50:50 PM: { Error: Syntax error: Expected corresponding JSX closing tag for <name>
5:50:50 PM:   1590 | <li parentName="ul" {...{"className":"g-type-long-body"}}>
5:50:50 PM:   1591 | <a parentName="li" {...{"id":"extravolumes-2","className":"__target-lic","aria-hidden":""}}></a>
5:50:50 PM: > 1592 | <p parentName="li"><a parentName="p" {...{"href":"#extravolumes-2","aria-label":"extravolumes permalink","className":"__permalink-lic"}}><inlineCode parentName="a">{`extraVolumes ((#v=terminatinggateways-defaults-extraVolumes)) (`}</inlineCode></a>{`array: []`}<inlineCode parentName="p">{`) - A list of extra volumes to mount. These will be exposed to Consul in the path `}</inlineCode>{`/consul/userconfig/`}<name>{`/\`.`}</p>
5:50:50 PM:        |                                                                                                                                                                                                                                                                                                                                                                                                                                         ^
5:50:50 PM:   1593 | <pre parentName="li" {...{"className":"language-yaml"}}><code parentName="pre" {...{"className":"language-yaml"}}><span parentName="code" {...{"className":"token key atrule"}}>{`extraVolumes`}</span><span parentName="code" {...{"className":"token punctuation"}}>{`:`}</span>{`
5:50:50 PM:   1594 |   `}<span parentName="code" {...{"className":"token punctuation"}}>{`-`}</span>{` `}<span parentName="code" {...{"className":"token key atrule"}}>{`type`}</span><span parentName="code" {...{"className":"token punctuation"}}>{`:`}</span>{` `}<span parentName="code" {...{"className":"token string"}}>{`'secret'`}</span>{`
5:50:50 PM:   1595 |     `}<span parentName="code" {...{"className":"token key atrule"}}>{`name`}</span><span parentName="code" {...{"className":"token punctuation"}}>{`:`}</span>{` `}<span parentName="code" {...{"className":"token string"}}>{`'my-secret'`}</span>
```

Reply:
The variable name was missing a backtick.